tests/behavior/ingamepenalty_attacking.lua

```diff
@@ -20,7 +20,7 @@ function on_goal()
   print("Ball: " .. inspect(state.ball))
   print("Ball was at x: " .. state.ball.position[1] .. " y: " .. state.ball.position[2])
   state.ball = nil
-  game_end_time = state.cycle_count + 200
+  state.finished = true
```

Comment:
While this fixes the crash, I would prefer if we simulated the game controller correctly.
So instead of aborting the simulation immediately prior to the crash, the game controller state should be set to ready after a goal was scored.

Reply:
Done, next iteration this should be moved into a helper function to be shared across lua files